Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2.yml

```diff
@@ -632,22 +648,6 @@ tasks:
               right:
                 value:
                   simple: Microsoft Word
-            - operator: isEqualString
```

Comment:
This part is a follow up of a previous bug that I noticed while working on this - in https://github.com/demisto/etc/issues/17351.
Though there is a condition prior to this task that was fixed, I realised that in a multi-file scenario this task might still be reached, and so this task had to be changed as well